content/en/agent/faq/linux-agent-2022-key-rotation.md

```diff
@@ -0,0 +1,124 @@
+---
+title: 2022 Linux Agent Key Rotation
+kind: faq
+---
+
+As a common best practice, Datadog periodically rotates the keys and certificates used to sign Datadog's Agent packages. The following GPG keys, used to sign the Agent RPM and DEB packages, reach their end-of-life in June 2022 and will be rotated in April 2022:
+
+- The RPM signing key with hash [`A4C0B90D7443CF6E4E8AA341F1068E14E09422B3`][1] will be rotated in April 4th 2022 and will be replaced by the key with hash [`C6559B690CA882F023BDF3F63F4D1729FD4BF915`][2]
+- The DEB signing key with hash [`A2923DFF56EDA6E76E55E492D3A80E30382E94DE`][3] will be rotated in April 18th 2022 and will be replaced by the key with hash `D75CEA17048B9ACBF186794B32637D44F14F620E`[4]
+
+Customers using Datadog's RPM or DEB packages might require a manual action to import the new key on their systems in order to install or upgrade the Agent after the rotation takes place.
+
+<div class="alert alert-info">
+**Note**: This DOES NOT affect the functionality of already running Agents, and only limits the ability to install or upgrade to a newer version of the Agent. This also doesn't affect Windows or MacOS Agents, nor Dockerized Linux Agents.
+</div>
+
+If you're using one of the following install methods, your hosts trust the key automatically and no further action is needed:
+
+* [Agent install script][5] v1.5.0 (released Jun 29, 2021)
+* [Chef cookbook][6] v4.11.0 (released Aug 10, 2021)
+* [Ansible role][7] v4.9.0 (released May 6, 2021)
+* [Puppet module][8] v3.13 (released Aug 11, 2021)
+* [SaltStack formula][9] v3.4 (released Aug 12, 2021)
+* [Elastic Beanstalk][10] config templates updated as of Mar 29, 2021 (should contain `DATADOG_RPM_KEY_FD4BF915.public` under `gpgkey`)
+* Containerized Agents (Docker/Kubernetes): No action needed on any version
+* Heroku: No action needed on any version.
+* Windows/MacOS Agents: No action needed on any version.
+
+Additionally, if you are using the DEB version of the Agent version 7.31.0 or greater, your hosts should have installed the `datadog-signing-keys` package that automatically trusts the new key.
```

Comment:
Why not add this as a bullet point to the list?

Reply:
Because it overlaps: you can be using an older chef but a newer Agent, or the other way around, and still be fine. How would you include it in the list above in a way that the overlap is clear?

Reply:
That's a fair point, we can leave it as a separate sentence. I didn't get/think about the overlap when reading it like this, but probably that is just me